script.js

```diff
@@ -0,0 +1,143 @@
+"use strict";
+const elements = {
+    toggleThemeBtn: document.querySelector(".toggle-theme"),
+    modeIcon: document.querySelector(".mode-icon"),
+    definitionsList: document.querySelector(".definitions"),
+    fonts: document.querySelector(".fonts"),
+    body: document.body,
+    dropdown: document.querySelector(".custom-dropdown"),
+    button: document.querySelector(".custom-dropdown .dropdown-button"),
+    options: document.querySelector(".custom-dropdown .dropdown-options"),
+    searchBtn: document.querySelector(".searchBtn"),
+    wordInput: document.querySelector(".wordInput"),
+    phonetic: document.querySelector(".phonetic"),
+    searchedWord: document.querySelector(".searchedWord"),
+    partOfSpeech: document.querySelector(".partOfSpeech"),
+    verbs: document.querySelector(".verbs"),
+    synonyms: document.querySelector(".synonym-word"),
+    noInput: document.querySelector(".no-input"),
+    errorTitle: document.querySelector(".errorTitle"),
+    errorMessage: document.querySelector(".errorMessage"),
+    dictionaryContainer: document.querySelector(".dictionary"),
+    errorContainer: document.querySelector(".errorContainer"),
+    source: document.querySelector(".link"),
+    option: document.querySelector(".option"),
+    playButton: document.getElementById("play-audio"),
+    audio: new Audio()
+};
+
+// Toggle between light and dark mode
+elements.toggleThemeBtn.addEventListener("change", (event) => {
+    if (event.target.checked) {
+        elements.body.style.setProperty("--bg-color", "var(--bg-color-dark)");
+        elements.body.style.setProperty("--text-color", "var(--text-color-dark)");
+        elements.body.style.setProperty("--search-bg", "var(--search-bg-dark)");
+        elements.body.style.setProperty(" --box-shadow", "var(--shadow-dark)");
+    } else {
+        elements.body.style.setProperty("--bg-color", "var(--bg-color-light)");
+        elements.body.style.setProperty("--text-color", "var(--text-color-light)");
+        elements.body.style.setProperty("--search-bg", "var(--search-bg-light)");
+        elements.body.style.setProperty(" --box-shadow", "var(--shadow-light)");
+    }
+});
+
+// Function to fetch dictionary data
+async function fetchDefinition(word) {
+    try {
+        // Fetch the response
+        const response = await fetch(`https://api.dictionaryapi.dev/api/v2/entries/en/${word}`);
+        
+        if (!response.ok) {
+            const error = await response.json();
+            handleError(error);
+        }
+
+        // Parse the response JSON
+        const data = await response.json();
+        console.log(data);
+        
+        return displayDefinition(data);
+
+    } catch (error) {
+        // console.log("Error:", error.message);
```

Comment:
Dont leave unnecessary comments , Always check and clean your code

Reply:
Yes sir